cartridges/app_storefront_custom/cartridge/static/default/css/experience/components/commerceAssets/facebookPost.css

```diff
@@ -0,0 +1,4 @@
+iframe {
```

Comment:
Even if you will only load the css in one file, it is a good practice to use some more specific css selectors, so that you make sure, that these styles won't be applied globally or affect some other iframes.

Reply:
Done